src/app/my-stories/page.tsx

```diff
@@ -0,0 +1,539 @@
+            <Card className='w-full max-w-md'>
+              <CardHeader className='text-center'>
+                <CardTitle>Authentication Required</CardTitle>
+                <CardDescription>Please log in to view your stories and sessions</CardDescription>
+              </CardHeader>
+              <CardContent className='text-center'>
+                <p className='text-sm text-muted-foreground'>
+                  Use the login button in the top navigation to sign in with Life Science AAI.
+                </p>
+              </CardContent>
+            </Card>
+          </div>
+        </Main>
+      </div>
+    );
+  }
+
+  return (
+    <div className='flex flex-col h-screen'>
+      <Header>My Stories</Header>
+      <Main>
+        <div className='space-y-6'>
+          <div className='flex items-center justify-between'>
+            <div>
+              <h1 className='text-2xl font-bold'>My Stories</h1>
+              <p className='text-muted-foreground'>Manage your sessions and shared stories</p>
+            </div>
+            <div className='flex gap-2'>
+              <Button
+                variant='destructive'
+                onClick={handleDeleteAllClick}
+                disabled={myStories.loading || (myStories.sessions.length === 0 && myStories.states.length === 0)}
+                className='flex items-center gap-1'
+              >
+                <AlertTriangle className='h-4 w-4' />
+                Delete All
+              </Button>
+              <Button onClick={myStories.loadAllData} disabled={myStories.loading}>
+                {myStories.loading ? 'Refreshing...' : 'Refresh'}
+              </Button>
+            </div>
+          </div>
+
+          {myStories.error && (
+            <div className='bg-destructive/15 text-destructive px-4 py-3 rounded-lg'>
+              <strong className='font-medium'>Error: </strong>
+              {myStories.error}
+            </div>
+          )}
+
+          <Tabs defaultValue='my-content' className='w-full'>
+            <TabsList className='grid w-full grid-cols-3'>
+              <TabsTrigger value='my-content'>My Content</TabsTrigger>
+              <TabsTrigger value='public-content'>Public Content</TabsTrigger>
+              <TabsTrigger value='storage'>Storage</TabsTrigger>
+            </TabsList>
+
+            <TabsContent value='my-content' className='space-y-6'>
+              <div className='grid gap-6 md:grid-cols-2'>
+                <div>
+                  <h2 className='text-xl font-semibold mb-4 flex items-center gap-2'>
+                    <FileText className='h-5 w-5' />
+                    My Sessions ({myStories.sessions.length})
+                  </h2>
+                  {myStories.loading ? (
+                    <div className='text-center py-8 text-muted-foreground'>Loading sessions...</div>
+                  ) : myStories.sessions.length === 0 ? (
+                    <Card>
```

Comment:
Let's render this as a full width table instead and add simple filter on title and description?